### ISSUE TITLE
Adding a Leaderboard

### ISSUE THREAD

**Issue by Kevinm27** [Kevinm27/StarShift#2]
I could add a leaderboard into the game, in addition I would like to add to the score system so it's not as brutal. It resets the score multiplier every time you are hit so I would like to do something about it. The leaderboard seems hard to implement since it will involve opening a file, adding to it and grabbing information kind of like a database. I would have to add a separate class that would take in the information and then add a few graphics screens to show the scores. In the end it is possible to add the leaderboard with a lot of time and patience.

**Comment by onzfonz** [comment]
Yeah but reading from files helps us make thinga more flexible. You could even do something like that for difficulty
